source/common/network/connection_impl.h

```diff
@@ -123,6 +128,8 @@ class ConnectionImpl : public virtual Connection,
   TransportSocketPtr transport_socket_;
   FilterManagerImpl filter_manager_;
   ConnectionSocketPtr socket_;
+  Envoy::RequestInfo::FilterStateImpl per_connection_state_;
```

Comment:
nit: this doesn't need Envoy::

Same in other places in this PR

Reply:
done